nav2_mppi_controller/src/critics/cost_critic.cpp

```diff
@@ -119,6 +122,13 @@ void CostCritic::score(CriticData & data)
     return;
   }
 
+  geometry_msgs::msg::Pose active_goal;
+  if (enforce_path_inversion_) {
```

Comment:
I think we can abstract this even more to a util:

```
geometry_msgs::msg::Pose getCriticGoal(data, enforce_path_inversion)
{
  if (enable_path_inversion) {
    return getLastPathPose(data.path);
  } else {
    return data.goal;
  }
}
```

that would make that function testable as a unit and we don't have all the missing coverage warnings in the individual critics for this block since we'd just call that one util in each to return the goal extracted